pyEpiabm/pyEpiabm/tests/test_func/test_closure_functional.py

```diff
@@ -54,7 +54,9 @@ def test_closure_present(self, mock_read, mock_csv):
             "test_input.csv", self.sim_params, self.file_params,
             HelperFunc.sweep_list_initialise())
 
-        mock_read.assert_called_with('test_input.csv')
+        mock_read.assert_called_with('test_input.csv',
```

Comment:
So in general any repetition of long(-ish) parameter chains like this is bad practice - if we were to chain one of these argument types later then we would have to go through and change all five appearances of that in this file. It also makes the code longer and less readable. Given the same arguments are repeated five times, it would be better to save this as a list in the setUp method (`self.read_params = [...]), and then just check mock_read was called with that list each time throughout the file. Does that make sense?

Reply:
Ditto in other files too - sadly I think no easy way to avoid duplication across files.

Reply:
All sorted